CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Changelog
 [v0.7.2](https://github.com/rigetti/forest-benchmarking/compare/v0.7.1...master) (in development)
 ------------------------------------------------------------------------------------
 
+### Announcements
+
+- Upgraded pyQuil to v3 (gh-215). As a result, the minimum supported Python version is now 3.7.
```

Comment:
Seems like a pretty significant change to usage, maybe this should be 0.8.0 now?

Reply:
Great call! Will adjust

Reply:
Done